src/main/java/blackjack/domain/machine/Record.java

```diff
@@ -0,0 +1,68 @@
+package blackjack.domain.machine;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+import blackjack.domain.participant.Dealer;
+import blackjack.domain.participant.Participant;
+import blackjack.domain.participant.Player;
+
+public enum Record {
+    VICTORY("승", 1, "패"),
+    DRAW("무", 0, "무"),
+    DEFEAT("패", -1, "승");
+
+    private final String playerRecord;
+    private final int recordNumber;
+    private final String dealerRecord;
+
+    Record(String playerRecord, int recordNumber, String dealerRecord) {
+        this.playerRecord = playerRecord;
+        this.recordNumber = recordNumber;
+        this.dealerRecord = dealerRecord;
+    }
+
+    public static Record getRecord(Player player, Dealer dealer) {
+        if (isBurst(player, dealer)) {
+            return getRecordForBurst(player, dealer);
+        }
+
+        return getOrdinaryRecord(player, dealer);
+    }
+
+    private static Record getOrdinaryRecord(Player player, Dealer dealer) {
+        return Objects.requireNonNull(Arrays.stream(Record.values())
+                .filter(record
+                        -> record.recordNumber == compare(player, dealer))
+                .findFirst()
+                .orElse(null));
+    }
+
+    private static int compare(Participant participant1, Participant participant2) {
+        return Integer.compare(participant1.score(), participant2.score());
+    }
+
+    private static boolean isBurst(Player player, Dealer dealer) {
+        return player.isBurst() || dealer.isBurst();
+    }
+
+    private static Record getRecordForBurst(Player player, Dealer dealer) {
+        if (dealer.isBurst() && player.isBurst()) {
+            return DEFEAT;
+        }
```

Comment:
```suggestion
        if (player.isBurst()) {
            return DEFEAT;
        }
```
로직을 간소화하면서 좀 더 명확하게 룰을 드러낼 수 있을 거 같네요 😀

Reply:
앗 감사합니다 !!